integrations/tensorflow/e2e/keras/iree_vision_test_suite.bzl

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Macro for building e2e keras vision model tests."""
+
+load("//bindings/python:build_defs.oss.bzl", "iree_py_test")
+
+def iree_vision_test_suite(
+        name,
+        configurations,
+        external_weights = None,
+        deps = None,
+        tags = None,
+        size = "large",
+        python_version = "PY3",
+        **kwargs):
+    """Creates one iree_py_test per configuration tuple and bundles them.
+
+    Args:
+      name: name of the generated test suite.
+      configurations: a list of tuples of (dataset, include_top, model,
```

Comment:
I agree that that would be more readable in some ways, but it would also take this section from 15 lines to 90 lines if se expand it legibly, e.g.

```
{
    dataset: "cifar10",
    include_top: 1,
    model_name: "MobileNet",
    backends: ["tf", "iree_vmla"],
},
```

Reply:
Fair enough. The right solution is really to expand the matrix, so this can wait

Reply:
Agreed. Sounds good!